lib/typeform_data/requestor.rb

```diff
@@ -4,6 +4,8 @@
 module TypeformData
   module Requestor
 
+    RETRY_EXCEPTIONS = [Net::HTTPServiceUnavailable, Errno::ECONNREFUSED].freeze
```

Comment:
The simplest would probably just be to make a list of exceptions to retry an optional parameter for initializing a `TypeformData::Client`. 

I'm not going to add that in this PR, though you're welcome to do!